tests/nfs3-readdirplus/test.yaml

```diff
@@ -0,0 +1,26 @@
+# *** Add configuration here ***
+
+args:
+- -k none
+
+checks:
```

Comment:
The goal here would be to test for some nfs logging fields related to readdirplus records.